docs/server_api/graphql/documentation.rst

```diff
@@ -0,0 +1,51 @@
+=====================
+Using the GraphQL API
+=====================
+
+This provides the documentation on how to use the GraphiQL client to query Bodhi's resources.
+
+The Basics
+==========
+
+The GraphQL API assists in getting data in and out of* Bodhi. It's an HTTP-based API that can be used to query & post data. It's composed of:
+
+- **Node**  : An individual object. (Eg: Release, Update, etc.)
+- **Edge**  : A connection between a collection of objects and a single object. (Eg: Releases in an Update)
+- **Field** : Data/attributes of an object. (Eg: id_prefix of a Release)
+
+Using the GraphiQL Client
+-------------------------
+
+GraphiQL is a GUI for editing and testing GraphQL queries and mutations cleanly.
+
+On the left side of the interface, you can enter traditional GraphQL queries. On the right tab, you will see data rendered as per the query.
+
+GraphQL Query Structure
+-----------------------
+
+A typical GraphQL query looks like this::
+
+{ nameOfQuery(optionalArgument: argumentValue){
+    fieldDesired
+  }
+}
+
+A typical GraphQL data response looks like this::
+
+{ "data":{
+    "nameOfQuery": [
+      {
+        "fieldDesired": value
+      }
+    ] 
+  }
+}
+
+Available Queries
+-----------------
+
+``nameOfQuery`` can take different values like ``allReleases``, ``getRelease``, and ``getUpdate``.
+
+:allReleases              Query all releases in Bodhi.
+:getRelease               Query specific releases using ``id_prefix``, ``name``, ``composedByBodhi``, and ``state`` as arguments.
+:getUpdate                Query specific updates using ``stableKarma``, ``unstableKarma``, ``stableDays``, ``updateType`` (Update.type), ``status``, ``request``, ``severity``, ``pushed``, ``critpath``, ``dateApproved``, ``alias``, ``releaseId``, ``userId``, ``testGatingStatus``, ``fromTag``
```

Comment:
Let's add an example section and put an example for each query using curl something like

```
curl -H 'Content-Type: application/json' -X POST -d '{"query": "query {allReleases {name state}}"}' https://bodhi.fedoraproject.org/graphql
```